### ISSUE TITLE
Add verbosity TVAE (progress bar + save the loss values)

### ISSUE THREAD

**Issue by npatki** [sdv-dev/CTGAN#300]
### Problem Description
Similar to #298, we should also add a verbosity option for TVAE that shows the user the fitting progress per epoch. After fitting, it should be possible to see the loss values.

### Expected behavior
1. Add a `verbose` parameter to TVAE (just like CTGAN). By default it can be `False`
2. When set to `True`, increment a progress bar per (epoch x data iteration). Also show/refresh the loss value, similar to #298 (except there is only 1 loss value)
3. Always save the loss values that correspond to the last fitting iteration. The columns can now be `Epoch`, `Batch` and `Loss`.

```python
>>> model = TVAE(verbose=True, epochs=100)
>>> model.fit()
Fitting TVAE ...
Loss 0.024 |████████████████████████████         | 750/1000 [00:33<00:10, 229.00it/s]

# after modeling
>>> model.loss_values
   Epoch     Batch    Loss 
0      1     1        1.7863
1      1     2        1.5484
2      1     3        1.3633
```

**Comment by TkinterinShanghai** [comment]
Did you remove this in `TVAESynthesizer`? This is probably quite a useful feature

**Comment by npatki** [comment]
Hi @TkinterinShanghai nothing should be removed in `TVAESynthesizer`. If you are having any problems, would you be able to [file a new issue](https://github.com/sdv-dev/SDV/issues) describing the code you're trying, errors you are seeing, etc.

**Comment by TkinterinShanghai** [comment]
According to the [docs](https://docs.sdv.dev/sdv/single-table-data/modeling/synthesizers/tvaesynthesizer) there is no `verbose` option. It doesnt seem to be present in the code either, based on what I saw in the docstring

**Comment by npatki** [comment]
Hi @TkinterinShanghai, I see what's going on.

We are currently discussing an issue in the _CTGAN_ library, which implements some raw ML algorithms for synthetic data (such as CTGAN and TVAE). You are referring to API and documentation in the _SDV_ library, which wraps around these algorithms (`TVAESynthesizer`)

If you use the [code within this current repo](https://github.com/sdv-dev/CTGAN/blob/main/ctgan/synthesizers/tvae.py#L105), the verbose option is available. However, the `TVAESynthesizer` is a wrapper in a different library, where we haven't yet exposed the `verbose` parameter. I will add a feature request for it in SDV.

**Comment by TkinterinShanghai** [comment]
Awesome, thanks!
